assets/css/rtl.scss

```
@@ -1,5 +1,5 @@
 // Algolia search
-.algolia-wrapper {
+.algolia-wrapper.rtl {
```

You can directly use the setting from the HTML tag
```suggestion
html[dir='rtl'] .algolia-wrapper {
```

---

I'll apply that. Didn't know about this feature